### PR TITLE
fix bug: add _IP/_HOST in principal of kerberos

### PR DESCRIPTION
**Pull request by zhaidongbo**

### CODE REVIEW

File: fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java [hunk 1]
```diff
@@ -91,6 +82,25 @@ public FileSystemManager() {
         writeBufferSize = BrokerConfig.hdfs_write_buffer_size_kb << 10;
         handleManagementPool.schedule(new FileSystemExpirationChecker(), 0, TimeUnit.SECONDS);
     }
+
+    private static String preparePrincipal(String originalPrincipal) throws UnknownHostException {
+        String finalPrincipal = originalPrincipal;
+        String[] components = originalPrincipal.split("[/@]");
+        if (components != null && components.length == 3) {
+            if (components[1].equals("_HOST")) {
+                // Convert hostname(fqdn) to lower case according to SecurityUtil.getServerPrincipal
```
Comment (wuyunfeng) [inline fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java]: delete unuseful comment?

Comment (wuyunfeng) [inline fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java]: delete unuseful comment?

**Review by wuyunfeng — CHANGES_REQUESTED** [review]

**Review by morningman — CHANGES_REQUESTED** [review]
Please change your commit log. You should capitalize the first letter. And I think this is a feature, not a bug?
`Support _IP/_HOST in principal of kerberos` should be better?

File: fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java (hunk 1 above)
Comment (zhaidongbo) [reply fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java]: I don't  have an environmental to test the case with the principal of the hostname(fqdn), so I think it's better to keep the comment for reminders.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**